constants.js

```diff
@@ -80,6 +80,7 @@ var csts = {
   DEBUG                   : process.env.PM2_DEBUG || false,
   WEB_IPADDR              : process.env.PM2_API_IPADDR || '0.0.0.0',
   WEB_PORT                : parseInt(process.env.PM2_API_PORT)  || 9615,
+  WEB_STRIP_ENV_VARS      : process.env.PM2_WEB_STRIP_ENV_VARS || true,
```

Comment:
I'm not sure about the default value for this, might break some implementations. WDYT @vmarchaud, shouldn't we set this to `false` by default so that things remain unchanged?